specification/deviceupdate/data-plane/Microsoft.DeviceUpdate/stable/2022-10-01/deviceupdate.json

```diff
@@ -2222,7 +2210,7 @@
             "$ref": "#/parameters/InstanceIdParameter"
           },
           {
-            "$ref": "#/parameters/OperationIdParameter"
```

Comment:
Is this change intended?

Reply:
Yes, we want to make sure the x-ms-client-name "logCollectionId" is used here as well.

Reply:
I checked and this feels correct indeed